backend/processing/check_files_checksums/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:18.04
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt update && apt install -y python3
+COPY task.py /opt/task.py
+RUN chmod +x /opt/task.py
+ENTRYPOINT ["python3", "/opt/task.py"]
```

Comment:
If you add a shebang line like `#!/usr/bin/env python3` you can simplify this to just `ENTRYPOINT ["/opt/task.py"]`.

Reply:
Done in d1cf6350db2a444ec180d0f6eec2e2bbd95f5062 . Thanks, I was missing shebang in one script.